### PR TITLE
[13.0][ADD] sale_order_line_delivery_state

### PR DESCRIPTION
**Pull request by JordiMForgeFlow**
Module based on sale_order_delivery_state, aimed at being able to have delivery state at sales order line level.

### CODE REVIEW

File: sale_order_line_delivery_state/models/sale_order_line.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            )
+            >= 0
+        )
+
+    def _partially_delivered(self):
+        """
+        Returns True if line has qty_delivered != to 0 and < ordered
+        quantities.
+
+        If `delivery` module is installed, ignores the lines with delivery costs
```
Comment (AdriaGForgeFlow) [inline sale_order_line_delivery_state/models/sale_order_line.py]: Nitpicking: I'm not sure this comment goes here or rather in the _compute_sale_line_delivery_state function where you actually filter out delivery costs

**Review by AdriaGForgeFlow — APPROVED** [review]
Functional and code review :+1:

File: sale_order_line_delivery_state/models/sale_order_line.py (hunk 1 above)
Comment (JordiMForgeFlow) [reply sale_order_line_delivery_state/models/sale_order_line.py]: Yes! I modified how the delivery lines were treated at one point and forgot to fix the comments. Thank you for spotting it! :)

**Review by AaronHForgeFlow — APPROVED** [review]
Functional + code :+1: 

I have a question. For services the status is not delivered, I guess this is intended right?
image

**Comment by JordiMForgeFlow** [comment]
> Functional + code +1
> 
> I have a question. For services the status is not delivered, I guess this is intended right? image

Exactly! As there is no delivery expected IMO it makes more sense to have this value rather than unprocessed, which denotes that a delivery needs to be processed at some time.

**Comment by AaronHForgeFlow** [comment]
Ok! I agree.

**Comment by JordiMForgeFlow** [comment]
@JordiBForgeFlow @pedrobaeza could I get some reviews here? :)

**Comment by pedrobaeza** [comment]
I don't use this module, but I trust in the reviewers, so

/ocabot merge nobump

**Pull request merged**